addons/mail/static/src/models/messaging.js

```diff
@@ -350,6 +350,13 @@ registerModel({
             isCausal: true,
             readonly: true,
         }),
+        /**
+         * Determines whether audio should be disabled. Usefull to prevent
+         * `NotAllowedError` in tests.
+         */
+        disableAudio: attr({
+            default: false,
+        }),
```

Comment:
IMO this field is poorly name. "disable audio" looks like an action, instead of a state.